### ISSUE TITLE
add options to `no-construct-stack-suffix`

### ISSUE THREAD

**Issue by ren-yamanashi** [ren-yamanashi/eslint-cdk-plugin#124]
### Feature Description

The `no-construct-stack-suffix` forces the Construct ID not to have a `construct` or `stack` suffix, but there are use cases where ‘only the construct suffix is allowed’ or ‘only the stack suffix is allowed’.  
For this reason, add the following options

```js
disallowedSuffixes: {
  enum: ["Construct", "Stack"]
}
```

⬇️ example for eslint.config.js rule

```js
export default [
// Both prohibited (default)
{
  "rules": {
    "no-construct-stack-suffix": ["error"]
  }
}

// Only Construct suffixes are prohibited.
{
  "rules": {
    "no-construct-stack-suffix": ["error", { "disallowedSuffixes": ["Construct"] }]
  }
}

// Only Stack suffixes are prohibited.
{
  "rules": {
    "no-construct-stack-suffix": ["error", { "disallowedSuffixes": ["Stack"] }]
  }
}
]
```

### Invalid Code Examples

When `disallowedSuffixes` is "Construct"
```typescript
// ❌ 
const bucket = new Bucket(this, "BucketConstruct");
```

When `disallowedSuffixes` is "Stack"
```typescript
// ❌ 
new MyStack(this, "MyStack");
```

### Valid Code Examples

When `disallowedSuffixes` is "Construct"
```typescript
// ✅
new MyStack(this, "MyStack");
```

When `disallowedSuffixes` is "Stack"
```typescript
// ✅
const bucket = new Bucket(this, "BucketConstruct");
```

### Rule Type

problem (possible errors)

### Implementation Details

- [ ] This rule is related to AWS CDK best practices
- [x] I'm willing to submit a PR for this feature
